Healex.HL7v2Anonymizer/Healex.HL7v2Anonymizer.csproj

```diff
@@ -3,9 +3,11 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
+    <AssemblyName>anonymizer</AssemblyName>
```

Comment:
Fine with me. But we also have a [`CSV-Anonymizer`](https://github.com/healexsystems/Healex.CSV-Anonymizer) that might collide when we also apply similar changes. I would recommend to include "v2" somewhere. What do you guys think?